Library/Formula/rdup.rb

```diff
@@ -12,6 +12,10 @@ class Rdup < Formula
     sha256 "955fc1c20fd4d9daddef07f0cc4f8475f310040ed797ba0e00e1a76f91a74a46" => :mavericks
   end
 
+  option "without-test", "Skip build-time tests (not recommended)"
```

Comment:
Sorry, just don't see the practical need for it in a way that would be used for more than one or two formulae.